hassio/src/snapshots/hassio-snapshots.ts

```diff
@@ -66,6 +67,32 @@ export class HassioSnapshots extends LitElement {
     await this.fetchSnapshots();
   }
 
+  private _computeSnapshotContent = memoizeOne(
```

Comment:
No need to memoize this, we call it a lot with different params, so doesn't work

Reply:
Currently each row call it twice, and since it's ordered, it will work for call 2 on each